pkg/pillar/types/diskmetrics.go

```diff
@@ -13,33 +13,50 @@ import (
 
 // ReportDiskPaths Report disk usage for these paths
 var ReportDiskPaths = []string{
-	"/",
+	"/",       // overlayfs inside EVE
+	"/hostfs", // actual IMGx inside EVE
 	IdentityDirname,
 	PersistDir,
 }
 
 // ReportDirPaths  Report directory usage for these paths
 var ReportDirPaths = []string{
-	PersistDir + "/downloads", // XXX old to be removed
-	PersistDir + "/img",       // XXX old to be removed
-	PersistDir + "/containerd",
-	PersistDir + "/tmp",
+	PersistConfigDir,
+	PersistStatusDir,
+	CertificateDirname,
+	SealedDirName,
+	ClearDirName,
+	PersistDebugDir,
+	PersistInstallerDir,
+	IngestedDirname,
+	NewlogDir,
+	PersistDir + "/containerd", // Old location
+	PersistDir + "/tmp",        // Should not be used by anything
 	PersistDir + "/log",
-	PersistDir + "/newlog",
-	PersistDir + "/config",
-	PersistDir + "/status",
-	PersistDir + "/certs",
 	PersistDir + "/checkpoint",
+	PersistDir + "/containerd-system-root",
+	PersistDir + "/netdump",
+	PersistDir + "/pubsub-large",
+	PersistDir + "/reserved",
+	PersistDir + "/etcd-storage",
+	PersistDir + "/memory-monitor/output", // for memory monitor dumps
+	PersistDir + "/netdump",
```

Comment:
It seemse that netdump dir is mentioned twice

Reply:
Fixed.